app/src/main/res/values/strings.xml

```diff
@@ -130,6 +130,8 @@
     <string name="excel_tv_view_text">Excel file selected: </string>
     <string name="select_excel_file">Select Excel file</string>
 
+    <string name="error_path_not_found">Error Path is empty. Please select a file</string>
```

Comment:
This error is when we are able to find the file path. Can you change the message here to

"Sorry, we are unable to access the file at this location."